admin_manual/installation/letsencrypt.rst

```diff
@@ -0,0 +1,335 @@
+
+.. literalinclude:: examples/lets_encrypt/renew-cron.sh
+    :language: bash
+
+delete.sh
+~~~~~~~~~
+
+This script deletes an issued certificate.
+Use the ``list.sh`` script to list issued certificates.
+
+.. literalinclude:: examples/lets_encrypt/delete.sh
+    :language: bash
+
+register.sh
+~~~~~~~~~~~
+
+As an example, this script creates a certificate for following domain / sub-domains.
+You can add or remove sub-domains as necessary.
+Use your domain / sub-domain names.
+The first (sub)domain name used in the script is taken for naming the directories created by Certbot.
+
+.. note::
+   You can create different certificates for different sub-domains by creating different scripts.
+
+- mydom.tld
+- www.mydom.tld
+- sub.mydom.tld
+
+.. literalinclude:: examples/lets_encrypt/register.sh
+    :language: bash
+
+.. note::
+   You can enable the ``--dry-run`` option which does a test run of the client only.
+
+Create an SSL certificate
+-------------------------
+
+With all the scripts created, to create an SSL certificate, run the following command:
+
+::
+
+  sudo /etc/letsencrypt/register.sh
+
+After you run the script, you will see output similar to the following:
+
+::
+
+  Saving debug log to /var/log/letsencrypt/letsencrypt.log
+  Obtaining a new certificate
+  Performing the following challenges:
+  http-01 challenge for example.com
+  Using the webroot path /var/www/html for all unmatched domains.
+  Waiting for verification...
+  Cleaning up challenges
+  Running post-hook command: service apache2 reload
+
+  IMPORTANT NOTES:
+   1. Congratulations! Your certificate and chain have been saved at:
+      /etc/letsencrypt/live/example.com/fullchain.pem
+      Your key file has been saved at:
+      /etc/letsencrypt/live/example.com/privkey.pem
+      Your cert will expire on 2018-06-18. To obtain a new or tweaked
+      version of this certificate in the future, simply run certbot
+      again. To non-interactively renew *all* of your certificates, run
+      "certbot renew"
+   2. If you like Certbot, please consider supporting our work by:
+
+      Donating to ISRG / Let's Encrypt:   https://letsencrypt.org/donate
+      Donating to EFF:                    https://eff.org/donate-le
+
+You can see that the SSL certificate’s been successfully created, and that it will expire on 2018-06-18.
+
+Add/Remove domains from the certificate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to *add* a domain like ``test.mydom.tld`` to your certificate, just add the domain in the domain shell script above, re-run it and reload the web server config.
+This can be useful when migrating from a sub-directory to sub-domain access.
+
+If you want to *remove* a sub-domain like ``www.mydom.tld`` from your SSL certificate, delete the certificate with the ``delete.sh`` script and then set up a new one.
+
+.. note::
+   This also implies that you need to comment the ``include`` directive and follow the steps afterward.
```

Comment:
Proposal:
This also implies that you need to comment the ``include`` directive (see web server config) and follow the steps afterwards

Reply:
Nice spot on that one.